pkg/virt-api/webhooks/mutating-webhook/preset.go

```diff
@@ -177,6 +178,8 @@ func mergeDomainSpec(presetSpec *kubev1.DomainSpec, vmiSpec *kubev1.DomainSpec)
 		}
 		if reflect.DeepEqual(vmiSpec.Features, presetSpec.Features) {
 			applied = true
+		} else {
+			overrides = append(overrides, fmt.Errorf("spec.features: %v != %v", presetSpec.Features, vmiSpec.Features))
 		}
 	}
 	if presetSpec.Devices.Watchdog != nil {
```

Comment:
`Watchdog` seems to be the only one supported. So it would still be possible to set e.g. Disks (which we should allow, together with volumes) although they are not supported right now.  We can either introduce a `DevicesPreset` struct or start merging everything in `Devices`.